### PR TITLE
feat: Start associating packaging shapes images in the taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
### What
- Start associating packaging shapes images in the taxonomy
- Sometimes we have several shapes per entry:
  - in that case I've tried to take the most representative
  - in some cases, create subentries that are currently commented out
- I have added suggested materials per the official CITEO guidelines
  - They could be displayed on top of material suggestion lists, and possibly used for potential Robotoff questions

### CODE REVIEW

File: taxonomies/packaging_shapes.txt [hunk 1]
```diff
@@ -138,9 +139,13 @@ hu:Doboz, dobozok
 it:Scatola, scatole, astuccio, scatoletta
 nl:Doos, doosje, Dozen, Doosjes, Pakje, Pakjes
 pt:Caixa, caixas
-#suggest:packaging_materials:nl: en:paper, en:cardboard
+# suggest:packaging_materials:nl: en:paper, en:cardboard
 description:nl:Een veelal kartonnen balkvormige verpakking
 
+fr:box de pâtes
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:145]:
```suggestion
# <en:box
# fr:box de pâtes
```

File: taxonomies/packaging_shapes.txt [hunk 2]
```diff
@@ -151,8 +156,15 @@ pl:wytłaczanka
...
 wikidata:en:Q959287
-#suggest:packaging_materials:nl: en:cardboard
+# suggest:packaging_materials:nl: en:cardboard
+# suggest:category: en:eggs
 description:nl:Kartonnen of geschuimd kunststoffen doos met holtes om eieren in te verpakken zonder dat deze elkaar raken
+# image:fr:boite-boitier-4.svg
+
+<en:Box
+fr:Boîte à hamburger
+# suggest:category: en:burgers
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:166]:
```suggestion
# <en:Box
# fr:Boîte à hamburger
# suggest:category: en:burgers
```

File: taxonomies/packaging_shapes.txt [hunk 3]
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
...
 wikidata:en:Q80228
-#suggest:packaging_materials:nl: en:glass
+# suggest:packaging_materials:nl: en:glass
 description:nl:Een langgerekt, cilindrisch en meestal van glas vervaardigd vat met een nauwe hals die met een dop of kurk af te sluiten is
 
+# fr:bouteille de vin non gazeux
+# image:fr:bouteille-2.svg
+# suggest:category: en:non-sparkling wines
+
+fr:bouteille de jus
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:212]:
```suggestion
# fr:bouteille de jus
```

File: taxonomies/packaging_shapes.txt [hunk 4]
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
...
+# image:fr:bouteille-6.svg
+# suggest:category: en:juices
+
+#fr:bouteille de bière
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:216]:
```suggestion
# fr:bouteille de bière
```

File: taxonomies/packaging_shapes.txt [hunk 5]
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
...
+# image:fr:bouteille-8.svg
+# suggest:category: en:detergeants
+
+#fr:bouteille de liquide vaisselle
+# image:fr:bouteille-9.svg
+# suggest:category: en:detergeants
+
+
+
+fr:flasque
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:238]:
```suggestion
# image:fr:bouteille-1.svg
# suggest:category: en:beers

# <en:bottle
# fr:grande bouteille de soda
# image:fr:bouteille-7.svg
# suggest:category: en:sodas

# <en:bottle
# fr:bouteille compactable
# image:fr:bouteille-3.svg
# suggest:category: en:waters

# <en:bottle
# fr:bouteille de detergeant
# image:fr:bouteille-8.svg
# suggest:category: en:detergeants

# <en:bottle
# fr:bouteille pour liquide vaisselle
# image:fr:bouteille-9.svg
# suggest:category: en:detergeants



fr:flasque
```

File: taxonomies/packaging_shapes.txt [hunk 6]
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
...
+fr:brique à ouvrir par découpe
+# image:fr:brique-1.svg
+# suggest:category: en:milks
+
+fr:brique biseautée avec bouchon revissable
+# image:fr:brique-2.svg
+
+fr:brique plate avec bouchon revissable
+# image:fr:brique-3.svg
+
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:256]:
```suggestion
# <en:Brick
# fr:brique à ouvrir par découpe
# image:fr:brique-1.svg
# suggest:category: en:milks

# <en:Brick
# fr:brique biseautée avec bouchon revissable
# image:fr:brique-2.svg

# <en:Brick
# fr:brique plate avec bouchon revissable
# image:fr:brique-3.svg

```

File: taxonomies/packaging_shapes.txt [hunk 7]
```diff
@@ -277,6 +344,17 @@ hu:Tubus, tubusok, Cső, csövek, cső alakú
 nl:Tube, Tubes
 pt:Bisnaga, bisnagas, tubo de bisnaga, tubos de bisnagas
 describtion:nl:Een verpakking in kokervorm gemaakt van metaal of plastic gevuld met een halfvloeibare stof
+# image:fr:tube-1.svg 
+# suggest:packaging_materials: en:cardboard, en:metal, en:glass, en:plastic, en:others
+#associated_packagings: en:cap
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:349]:
```suggestion
# associated_packagings: en:cap
```

File: taxonomies/packaging_shapes.txt [hunk 8]
```diff
@@ -535,13 +666,15 @@ pt:Película, películas, filme, filmes, filme extensível, filme estirável
...
 th:ฟิล์ม
-ur:فلم‎
+ur:فلم
 sv:Film
 zh:菲林
 weight:en:light
 description:en:A thin layer of plastic or metal
 description:nl:Dun en buigzaam vel van een metaal of kunststof
-#comment:en:Some languages might distinghuish between plastic film and metal foil. Film in english can also mean a thin sheet of oil on water for instance.
+#comment:en:Some languages might distinghuish between plastic film and metal foil. Film in English can also mean a thin sheet of oil on water for instance.
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:675]:
```suggestion
# comment:en:Some languages might distinguish between plastic film and metal foil. Film in English can also mean a thin sheet of oil on water for instance.
```

File: taxonomies/packaging_shapes.txt [hunk 9]
```diff
@@ -611,149 +759,245 @@ hu:címke, címkék
...
-#suggest:packaging_materials:en: en:plastic, en:cardboard
+# suggest:packaging_materials:en: en:plastic, en:cardboard
+# image:fr:applicateur-plastique.svg (applicateur dévissable de flacon de sauce)
+# image:fr:applicateur-plastique-2.svg (tampon hygiénique)
 
 en:Bag-in-box
 fr:Bag-in-box, Cubitainer
 nl:Zak-in-doos
 wikidata:en:Q799810
+# made of: Boîte carton à recycler, poche plastique à jeter
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:776]:
```suggestion
# made of: fr:Boîte carton à recycler, fr:poche plastique à jeter
```

**Comment by teolemon** [comment]
@aleene @stephanegigandet would like your input

**Comment by stephanegigandet** [comment]
> @aleene @stephanegigandet would like your input

It's a great idea to add images for each packaging shape.

Regarding the format of the property:
- we should use image:xx: instead of image:en: so that the images apply to all languages
- we should move the CITEO images to from html/images/icons/packaging-shapes/ to html/images/icons/packaging-shapes/citeo/ so that we can easily add other images

so the format should be something like:

image:xx:/images/icons/packaging-shapes/citeo/brique-2.svg

Regarding the content:

We may not need to add all Citeo icons as entries in the taxonomy. If we do add them, then we should have an English names for them. It might be useful to have all of them at some point (e.g. to recognize them, even though my understanding is that they will go away in France).

The focus at first should be on getting an image for all existing entries (or their parents, we can inherit the property), so that we can use them in the packaging edit interface and in knowledge panels etc.
e.g. We can assign the most generic looking bottle image to en:bottle.

**Comment by teolemon** [comment]
@stephanegigandet 👍 
- fixed the xx lang code
- moved the images to a dedicated CITEO directory
- everything remains commented out for the time being, and we can uncomment as we feel confortable

**Review by stephanegigandet — APPROVED** [review]
Merging as-is, we can decide on the formatting, add image paths etc. after.

**Pull request merged**